Easydict/Swift/Feature/Configuration/Configuration+Defaults.swift

```diff
@@ -311,7 +311,32 @@ extension Defaults.Keys {
     static let aliAccessKeySecret = Key<String?>(EZAliAccessKeySecret)
 
     // Gemni
-    static let geminiAPIKey = Key<String?>(EZGeminiAPIKey)
+    static let geminiAPIKey = Key<String?>("EZGeminiAPIKey")
+    static let geminiTranslation = Key<String>(
+        translationStoredKey(.gemini),
+        default: "1"
+    )
+    static let geminiDictionary = Key<String>(
+        dictionaryStoredKey(.gemini),
+        default: "1"
+    )
+    static let geminiSentence = Key<String>(
+        sentenceStoredKey(.gemini),
+        default: "1"
+    )
+    static let geminiServiceUsageStatus = Key<ServiceUsageStatus>(
+        serviceUsageStatusStoredKey(.gemini),
+        default: .default
+    )
+    static let geminiModel = Key<String>("geminiModel", default: GeminiModel.gemini1_5_flash.rawValue)
+    static let geminiAvailableModels = Key<String?>(
+        "geminiAvailableModels",
```

Comment:
In order to implement this feature https://github.com/tisfeng/Easydict/issues/479 , we should use a dynamic variable to be used as the key, like the `serviceUsageStatusStoredKey(.gemini)` in the previous section, to make sure that all the services storing data have the same format for the key.

You can try this,  if can't, you can put it in and I'll do it later.

Reply:
OK, I will try this

Reply:
The feature is working, does the code meet the intended usage?

Reply:
It already works, I will optimize the code later.

Reply:
I mean I pushed a new commit on dynamic variable, I'm wondering if it meets the expectation for dynamic variable